test/compilable/dtoh_TemplateDeclaration.d

```diff
@@ -15,6 +15,7 @@ template <typename T>
 struct A
 {
     // Ignoring var x alignment 0
+public:
```

Comment:
Arguably redundant to have public on a struct.

Reply:
Yeah, that's a bug caused by the current template handling (gonna fix that in a later PR).